### ISSUE TITLE
Always analyses loss file even with nothing that needs it

### ISSUE THREAD

**Issue by RainehDaze** [Zyin055/Inspect-Embedding-Training#9]
Even if the file is edited to not check learning rate, and not output a loss graph, it will _still_ fail to proceed when asked to generate a file based off embeddings alone. Fairly simple idea--if it's not being asked to make a loss graph or output plots after generation, or include learning rate on the vector graph, _why is it always trying to analyse this file anyway?_

Which is a pain if the loss CSV _isn't_ available for any reason.

**Comment by Zyin055** [comment]
See if that newest update works for you.

**Comment by RainehDaze** [comment]
Looks like it does; it's pretty much what I was trying to do on my end (but having an annoying time because of whitespace issues). Thanks.
